tox.ini

```diff
@@ -32,9 +32,8 @@ minversion=2.4
 skip_missing_interpreters=True
 toxworkdir=build/
 envlist=lint, mypy, black,
-    apidocs,narrativedocs,newsfragment,
+    apidocs, narrativedocs, newsfragment,
     manifest-checker, twine,
-    py36-alldeps-nocov
```

Comment:
This removes the only unit test run on the default env list. Is that intentional?

Reply:
Does the default envlist has any usage? I saw twine in that list...I was thinking that this list is some legacy stuff```

```